metadata-ingestion/docs/sources/powerbi/powerbi_pre.md

```diff
@@ -103,3 +104,47 @@ combine_result
 By default, extracting endorsement information to tags is disabled. The feature may be useful if organization uses [endorsements](https://learn.microsoft.com/en-us/power-bi/collaborate-share/service-endorse-content) to identify content quality.
 
 Please note that the default implementation overwrites tags for the ingested entities, if you need to preserve existing tags, consider using a [transformer](../../../../metadata-ingestion/docs/transformer/dataset_transformer.md#simple-add-dataset-globaltags) with `semantics: PATCH` tags instead of `OVERWRITE`.
+
+## Admin Access vs Data Ingestion
+### Service Principal As Member In Workspace 
+If you have added Service Principal as `member` in workspace then PowerBI Source would be able ingest below metadata of that particular workspace 
+
+  - Dashboards 
+  - Reports 
+  - Dashboard's Tiles
+  - Report's Pages
+
+### Service Principal As Admin in Tenant Setting and Added as Member In Workspace
+If you have added Service Principal as `member` in workspace and also allowed below permissions  
+
+  - Allow service principal to use read-only PowerBI Admin APIs
+  - Enhance admin APIs responses with detailed metadata
+
+PowerBI Source would be able to ingest below listed metadata of that particular workspace 
+
+  - Lineage 
+  - PowerBI Dataset 
+  - Endorsement as tag
+  - Dashboards 
+  - Reports 
+  - Dashboard's Tiles
+  - Report's Pages
+
+### Service Principal As Admin in Tenant Setting
+In this scenario Service Principal is only added in tenant setting and not added in individual workspaces.
+
+Service Principal is allowed below permissions
+
+  - Allow service principal to use read-only PowerBI Admin APIs
+  - Enhance admin APIs responses with detailed metadata
+
+In PowerBI Source recipe add flag `admin_apis_only: true` to configure PowerBI Source to use PowerBI Admin APIs only to retrieve the meta-data. PowerBI Source would be able to ingest below listed metadata of all workspaces.
```

Comment:
to retrieve the meta-data -> to extract metadata

Reply:
done